### PR TITLE
global: license change and files cleanup

### PR DESCRIPTION
**Pull request by egabancho**

### CODE REVIEW

**Comment by egabancho** [comment]
Needs https://github.com/inveniosoftware/pytest-invenio/pull/23 and probably a new release of `pytest-invenio`

File: CHANGES.rst [hunk 1]
```diff
@@ -1,29 +1,14 @@
...
-
-    In applying this license, CERN does not
-    waive the privileges and immunities granted to it by virtue of its status
-    as an Intergovernmental Organization or submit itself to any jurisdiction.
 
 Changes
 =======
 
-Version 1.0.0a2 (released August 18, 2017)
+Version 1.0.0 (released 2018-11-26)
```
Comment (lnielsen) [inline CHANGES.rst]: I think the module still needs a review and e.g. API documentation is missing as far as I can see. So better just bump it to 1.0.0a3 or b1

**Comment by lnielsen** [comment]
Everything looks good besides the small comment on the version number.

File: CHANGES.rst (hunk 1 above)
Comment (egabancho) [reply CHANGES.rst]: Yeap, you are right, I got over exited with the change 😜

**Comment by lnielsen** [comment]
pytest-invenio was released and I reran the tests in travis. Seems like there's a docstrings test failure. @egabancho will you have a look?

**Comment by lnielsen** [comment]
Now it's just Python 3 vs 2 problems it seems like :-)

**Comment by egabancho** [comment]
Yeah, it kind of sucks because the exception raised inside the docs in python 2 is just the name and in python 3 is the full name of the exception class. I’ll check if there is a fix for this situation

**Comment by egabancho** [comment]
Now everything seems to be green 🎉 

I added [`doctest.IGNORE_EXCEPTION_DETAIL`](https://docs.python.org/3/library/doctest.html#doctest.IGNORE_EXCEPTION_DETAIL) to the test that was complaining, which ignores also the exception message, which removes a great part of the purpose of this kind of tests 😞  but the exception type is still checked.

**Pull request merged**